newfile

```diff
@@ -0,0 +1,3 @@
+Here's a new file, wahoo!
```

Comment:
(Comment on the full diff view) (Line retained)

Yay!

Reply:
blah

Reply:
blah